api/src/pcapi/routes/backoffice/templates/accounts/update_requests_list.html

```diff
@@ -0,0 +1,114 @@
+    <div>
+      {% if rows and rows.total > 0 %}
+        <div class="d-flex justify-content-between">
+          <p class="lead num-results">{{ rows.total }} dossier{{ rows.total | pluralize }}</p>
+          <div>{% include "components/search/pagination.html" %}</div>
+        </div>
+        <table class="table mb-4">
+          <thead>
+            <tr>
+              <th scope="col"></th>
+              <th scope="col">Dossier</th>
+              <th scope="col">État</th>
+              <th scope="col">Date des derniers messages</th>
+              <th scope="col">Demandeur</th>
+              <th scope="col">Jeune</th>
+              <th scope="col">Modification</th>
+              <th scope="col">Instructeur</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for update_request in rows.items %}
+              <tr>
+                <td></td>
+                <td>
+                  {% set link = "https://www.demarches-simplifiees.fr/dossiers/" + update_request.dsApplicationId|string %}
+                  <a href="{{ link }}"
+                     class="link-primary"
+                     target="_blank"
+                     title="Ouvrir le dossier DS">{{ update_request.dsApplicationId }} <i class="bi bi-box-arrow-up-right"></i></a>
+                </td>
+                <td>{{ update_request.status | format_dms_application_status }}</td>
+                <td>
+                  {% if update_request.dateLastUserMessage %}
+                    <span class="fw-bold">Demandeur :</span>
+                    <br />
+                    {{ update_request.dateLastUserMessage | format_date_time }}
+                    <br />
+                  {% endif %}
+                  {% if update_request.dateLastInstructorMessage %}
+                    <span class="fw-bold">Instructeur :</span>
+                    <br />
+                    {{ update_request.dateLastInstructorMessage | format_date_time }}
+                    <br />
+                  {% endif %}
+                </td>
+                <td>
+                  {{ update_request.firstName | empty_string_if_null }} {{ update_request.lastName | empty_string_if_null }}
+                  {% if update_request.birthDate %}
+                    <br />
+                    né(e) le {{ update_request.birthDate | format_date }} ({{ update_request.applicant_age }} ans)
+                  {% endif %}
+                  <br />
+                  {{ update_request.email }}
+                </td>
+                <td>
+                  {% if update_request.user %}
+                    {{ links.build_public_user_name_to_details_link(update_request.user) }}
+                    <br />
+                    né(e) le {{ update_request.user.birth_date | format_date }} ({{ update_request.user.age }} ans)
+                    <br />
+                    {{ update_request.user.email }}
+                  {% else %}
+                    Utilisateur non trouvé
```

Comment:
Suggestion, laissée à ton appréciation : 
```suggestion
                    Compte jeune non trouvé
```